datamodel/extra_scripts/postgresql/211_nhr_views.sql

```diff
@@ -0,0 +1,167 @@
+--- koppel nevenvestigingen met hoofdvestiging
+create or replace view v_kvk_hoofd_nevenvestiging as
+select 			s.naam as hoofdvestigingnaam,
+				coalesce(v.fk_15ond_kvk_nummer::text, v.fk_17mac_kvk_nummer::text) as kvknummer,
+				count(v2.sc_identif)  								 		       as aantal_nevenvestigingen,
+				array_to_string(array_agg(distinct s2.naam), ' | ')    			   as nevenvestigingen
+from vestg v
+join subject s on v.sc_identif = s.identif
+join vestg v2 on coalesce(v.fk_15ond_kvk_nummer, v.fk_17mac_kvk_nummer) = coalesce(v2.fk_15ond_kvk_nummer, v2.fk_17mac_kvk_nummer)
+join subject s2 on v2.sc_identif = s2.identif 
+where v.hoofdvestiging = 'Ja' and v2.hoofdvestiging = 'Nee'
+group by hoofdvestigingnaam, coalesce(v.fk_15ond_kvk_nummer::text, v.fk_17mac_kvk_nummer::text);
+
+-- koppel subject en vestg tabel met het adresseerbaarobject
+create materialized view mb_kvk_adres as 
+select 		
+			row_number() over ()     		 as objectid,
```

Comment:
dit is fout: zie https://b3partners.atlassian.net/browse/SUPPORT-14692

Reply:
aanpassing zojuist gedaan